Helper/DeviceView.php

```diff
@@ -66,11 +63,17 @@ class DeviceView
     protected $switchParam = self::SWITCH_PARAM_DEFAULT;
 
     /**
+     * @var array
+     */
+    protected $configRedirect;
+
+    /**
      * Constructor
      *
-     * @param \Symfony\Component\DependencyInjection\Container $serviceContainer
+     * @param RequestStack $requestStack
+     * @param array        $configRedirect
      */
-    public function __construct(RequestStack $requestStack = null)
+    public function __construct(RequestStack $requestStack = null, $configRedirect = null)
```

Comment:
The bundle configuration is injected into the contructor, i don't understand, what do you expect, can you explain ?